CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Update `buf beta price` with the latest pricing information.
 - Fix issue where `buf generate --exclude-path` was not properly excluding paths
   for remote modules.
+- Alias the `--draft` flag for `buf push` to `--branch` and update flag description.
```

Comment:
I would not call out the alias. More specifically, I would actually only do this when you are ready to deprecate `--draft` entirely, and then deprecate and hide `--draft`, and add `--branch.